tools/devops/automation/build-pipeline.yml

```diff
@@ -7,22 +7,27 @@
 #
 parameters:
 - name: runTests
+  displayName: Run Simulator Tests
   type: boolean
   default: true
 
 - name: runDeviceTests
+  displayName: Run Device Tests (includes macOS)
```

Comment:
It would be nice to be able to run only the macOS tests (or vice versa) - that's separate from this PR though.

Reply:
Will do.